kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/WorkflowRunner.kt

```diff
@@ -45,7 +50,7 @@ interface WorkflowRunner<out OutputT> {
    * A stream of the [output][OutputT] values emitted by the running
    * [Workflow][com.squareup.workflow.Workflow].
    */
-  val output: Observable<out OutputT>
+  val output: Flowable<out OutputT>
```

Comment:
Our API on this class is kind of a weird hybrid now, with a `ReceiveChannel` as its fundamental input but Rx objects as its output. Would it be too restrictive to stick with `Flowable` for input as well? Too awkward to use channels for the outputs?

Once Flow stabilizes, will we use it for output as well as input?

Reply:
Here's a thought: what if the WorkflowHost became a public member on the WorkflowRunner, with the Rx methods as conveniences? Could even extract them to a `workflow-ui-android-rx` module in a follow up, if that's not too fine grained.

Reply:
> Our API on this class is kind of a weird hybrid now, with a `ReceiveChannel` as its fundamental input but Rx objects as its output. Would it be too restrictive to stick with `Flowable` for input as well?

Oh yea, definitely.

> Once Flow stabilizes, will we use it for output as well as input?

Definitely.

> what if the WorkflowHost became a public member on the WorkflowRunner

`WorkflowHost`'s API is still a bit of a footgun, so I'd like to hold off for now.

Reply:
- [ ] remove channel public API